process/accounts/serialize.go

```diff
@@ -189,7 +189,7 @@ func prepareSerializedAccountBalanceHistory(
 	}
 
 	id += fmt.Sprintf("-%d", account.Timestamp)
-	meta := []byte(fmt.Sprintf(`{ "index" : { "_index":"%s", "_id" : "%s" } }%s`, index, id, "\n"))
+	meta := []byte(fmt.Sprintf(`{ "index" : { "_index":"%s", "_id" : "%s" } }%s`, index, converters.JsonEscape(id), "\n"))
```

Comment:
since you prepare the ID, sanitizing is not necessary. but it can remain like this in order to avoid programming errors that would generate non readable characters

Reply:
Exactly this is an extra protection